tests/generation/Bears.py

```diff
@@ -286,10 +287,27 @@ def test_filter_relevant_bears_gruntfile_present(self):
 
     def test_print_relevant_bears(self):
         with retrieve_stdout() as custom_stdout:
-            print_relevant_bears(self.printer, filter_relevant_bears(
-                [('Python', 70), ('Unknown', 30)], self.printer,
-                self.arg_parser, {}))
+            languages = [('Python', 70), ('Unknown', 30)]
+            bears_filtered = filter_relevant_bears(languages,
+                                                   self.printer,
+                                                   self.arg_parser, {})
+            relevant_bears = OrderedDict([('unusable', {}),
+                                          ('usable', bears_filtered)])
+
+            print_relevant_bears(self.printer, relevant_bears)
             self.assertIn("PycodestyleBear", custom_stdout.getvalue())
+            # Should print only the usable bears
+            self.assertNotIn("unusable",
+                             custom_stdout.getvalue())
+
+    def test_print_relevant_bears_no_bears(self):
+        with retrieve_stdout() as custom_stdout:
+            print_relevant_bears(self.printer, OrderedDict([('unusable', {}),
+                                                            ('usable', {})]))
+            self.assertNotIn("usable", custom_stdout.getvalue())
+            # Should print only the usable bears
```

Comment:
no output is being printed in this case right?
this comment seems misleading because of that.